Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.18.4-buster
```

Comment:
Buster seems super heavy to use for a prebuilt linux binary. Can we use alpine or something instead?